### ISSUE TITLE
Advanced Flare System for Minimap

### ISSUE THREAD

**Issue by simonsan** [SFTtech/openage#1274]
@HeavenlyChorus was proposing an addition to the Taunt/Flare System in SE discord that I like, hence I'm opening this issue, but feel free to add.

Clients could be given a feature to draw on the minimap and sent this drawing to their teammates where the drawing gets rendered on their minimap - as an adavanced flare.

The drawing should be automatically dismissed after three seconds.

In addition to that there could be a few custom taunts that help to describe/interpret the drawing (e.g.): 
110 - "I wall to here" 
111 - "I go raid you defend at home" 
112 - "quickwall this!"

**Comment by heinezen** [comment]
I think having an additional "bigger map" feature could help with that. Maybe the minimap could be displayed on the entire screen with the press of a button. This would make the things @HeavenlyChorus mentiones easier to draw.

**Comment by simonsan** [comment]
> I think having an additional "bigger map" feature could help with that. Maybe the minimap could be displayed on the entire screen with the press of a button. This would make the things @HeavenlyChorus mentiones easier to draw.

Indeed, we also already have that in [interface.md](https://github.com/SFTtech/openage/blob/master/doc/ideas/interface.md).
